Shopify/cli
# Releasing extensions binary in an automatic way

### WHY are these changes introduced?

- Releasing the extensions binary at the same time that the CLI release is published
- Syncing extension binary version with CLI version so both of the works with the same version number

WTB: 
- Move the `shipit.test`  to `shipit.production` 

### WHAT is this pull request doing?
- Created a release script:
  - Generates extension binaries for all the operating systems
  - Creates a new release in github using current cli-kit version
  - Upload binaries as release assets

### How to test your changes?
- Delete temporary [release 3.7.2_SNAPSHOT](https://github.com/Shopify/cli/releases) generated in previous testings
- Access [shipit test stack](https://shipit.shopify.io/shopify/cli/test) and push `Deploy` in the last commit
- Waits until the deploy finishes correctly
- Verify that the release [release 3.7.2_SNAPSHOT](https://github.com/Shopify/cli/releases) has been generated correctly
- Run the command `SHOPIFY_ENV=production yarn shopify app scaffold extension`. Using env variable SHOPIFY_ENV makes the CLI to download de extensions binary release with the cli-kit version number (the same generated in the previous steps)
